test/integration/commands/excel2json/test_new_lists.py

```diff
@@ -34,9 +36,87 @@ def test_second_list(expected_lists: list[dict[str, Any]], result_lists_correct:
+        "\n----------------------------\n"
+        "ID: 'duplicate_node_id'\n"
+        "    - Excel 'testdata/invalid-testdata/excel2json/new_lists_duplicate_node_ids/list_duplicate_node_ids.xlsx' "
+        "| Sheet 'duplicate_node_id1' | Row 3\n"
+        "    - Excel 'testdata/invalid-testdata/excel2json/new_lists_duplicate_node_ids/list_duplicate_node_ids.xlsx' "
+        "| Sheet 'duplicate_node_id2' | Row 3"
+    )
+    with pytest.raises(InputError, match=expected):
+        new_excel2lists(Path("testdata/invalid-testdata/excel2json/new_lists_duplicate_node_ids"))
+
+
+def test_duplicate_list_name() -> None:
+    expected = regex.escape(
+        "\nThe excel file(s) used to create the list section have the following problem(s):"
+        "\n\n---------------------------------------\n\n"
+        "The name of the list must be unique across all the excel sheets.\n"
+        "The following sheets have lists with the same name:\n"
+        "    - Excel file: 'testdata/invalid-testdata/excel2json/new_lists_duplicate_listname/list_duplicate_1.xlsx', "
+        "Sheet: 'duplicate_list', List: 'List 2'\n"
+        "    - Excel file: 'testdata/invalid-testdata/excel2json/new_lists_duplicate_listname/list_duplicate_2.xlsx', "
+        "Sheet: 'duplicate_list', List: 'List 2'"
+    )
+    with pytest.raises(InputError, match=expected):
+        new_excel2lists(Path("testdata/invalid-testdata/excel2json/new_lists_duplicate_listname"))
+
+
+def test_invalid_shape() -> None:
+    expected = regex.escape(
+        "\nThe excel file(s) used to create the list section have the following problem(s):"
+        "\n\n---------------------------------------\n\n"
+        "The Excel file "
+        "'testdata/invalid-testdata/excel2json/new_lists_invalid_shape/list_missing_translation_column.xlsx' "
+        "contains the following problems:\n\n"
+        "The excel sheet 'Sheet1' has the following problem(s):\n"
+        "    - missing translations: All nodes must be translated into the same languages. "
+        "Based on the languages used, the following column(s) are missing: de_list"
+    )
+    with pytest.raises(InputError, match=expected):
+        new_excel2lists(Path("testdata/invalid-testdata/excel2json/new_lists_invalid_shape"))
+
+
+def test_missing_translation() -> None:
+    expected = regex.escape(
+        "\nThe excel file(s) used to create the list section have the following problem(s):"
+        "\n\n---------------------------------------\n\n"
+        "The Excel file "
+        "'testdata/invalid-testdata/excel2json/new_lists_missing_translations/list_missing_translation_cell.xlsx' "
+        "contains the following problems:\n\n"
+        "The excel sheet 'missing_translation_cell' has the following problem(s):\n"
+        "In one list, all the nodes must be translated into all the languages used. "
+        "For the following nodes, the translations are missing:\n"
+        "    - Row Number: 3 Column(s): de_1"
```

Comment:
There should be a comma to separate the row number from the column
```suggestion
        "    - Row Number: 3, Column(s): de_1"
```